libs/common-services/src/main/java/uk/gov/di/ipv/core/library/exceptions/OverwriteAvoidedException.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.di.ipv.core.library.exceptions;
+
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+public class OverwriteAvoidedException extends Exception {
```

Comment:
I'd prefer `CredentialAlreadyExistsException` or something similar - feels like a more accurate description of the 'exception'